vorestation.dme

```diff
@@ -3968,9 +3968,9 @@
 #include "maps\southern_cross\items\clothing\sc_accessory.dm"
 #include "maps\southern_cross\items\clothing\sc_suit.dm"
 #include "maps\southern_cross\items\clothing\sc_under.dm"
-#include "maps\southern_cross\loadout\loadout_head.dm"
 #include "maps\southern_cross\loadout\loadout_suit.dm"
 #include "maps\southern_cross\loadout\loadout_uniform.dm"
+#include "maps\southern_cross\loadout\loadout_vr.dm"

```

Comment:
this could probably go where all the other loadout headgear is (I want to phase out the southern_cross files anyway)